spec/helpers/guess_helper_spec.rb

```diff
@@ -0,0 +1,47 @@
+require "spec_helper"
+
+RSpec.describe GuessHelper, type: :helper do
+  let(:target_word) { "word" }
+  let(:guessed_letters) { [] }
+
+  describe "generate_obfuscated_word" do
+    context "with no guesses" do
+      it "returns a string of padded underscores only" do
+        helper.generate_obfuscated_word(target_word, guessed_letters).should == " _  _  _  _ "
```

Comment:
`should ==` is old syntax, we should use `expect`:

```suggestion
        expect(helper.generate_obfuscated_word(target_word, guessed_letters)).to eq(" _  _  _  _ ")
```

Reply:
Addressed in 2877447 @trezona-lecomte